sqsjobs/item.go

```diff
@@ -148,7 +156,7 @@ func (i *Item) Ack() error {
 	return nil
 }
 
-func (i *Item) Nack() error {
+func CommonNack(i *Item) error {
```

Comment:
CommonNack should be private, e.g. : `commonNack`. In Go, public - uppercase, private - lowercase.